### PR TITLE
Cancellation of reservation

### LINKED ISSUE #60: Cancellation of reservation
Cancellation of reservation

### PR DESCRIPTION
**Pull request by Danil98Borisov**
Cancellation of reservation

### CODE REVIEW

File: src/app/components/user-booking/user-booking.component.ts [hunk 1]
```diff
@@ -32,6 +32,11 @@ export class UserBookingComponent implements OnInit {
     }
 
 }
+  public delete(reservation: Reservation): void {
```
Comment (AndrewSmrnv) [inline src/app/components/user-booking/user-booking.component.ts]: давай переименуем delete -> cancel

File: src/app/components/user-booking/user-booking.service.ts [hunk 2]
```diff
@@ -27,4 +27,13 @@ export class UserBookingService {
     );
   }
 
+  deleteReservation(reservation: Reservation){
```
Comment (AndrewSmrnv) [inline src/app/components/user-booking/user-booking.service.ts]: давай переименуем delete -> cancel

File: src/app/components/user-booking/user-booking.component.ts (hunk 1 above)
Comment (Danil98Borisov) [reply src/app/components/user-booking/user-booking.component.ts]: done

File: src/app/components/user-booking/user-booking.service.ts (hunk 2 above)
Comment (Danil98Borisov) [reply src/app/components/user-booking/user-booking.service.ts]: done

**Pull request merged**